src/ngraph/runtime/cpu/cpu_runtime_context.hpp

```diff
@@ -38,6 +38,7 @@ namespace ngraph
             struct CPURuntimeContext
             {
                 int64_t* op_durations;
+                bool* p_en;
```

Comment:
I'm all for short variable names, but I have no idea what this variable is or does. Can you name it more descriptively.